types/artifacts.d.ts

```diff
@@ -145,6 +145,7 @@ declare global {
             snippet: string;
             target: string[];
             failureSummary?: string;
+            title?: string;
```

Comment:
is there a different word we could use? `title` seems very particular to html rendering rather than describing what the value is...but I don't have a good suggestion. `snippet` is already taken :) `description`? `summary`? ...

Reply:
+1 to using a different name, I kept thinking this was the `title` attribute until I opened the PR 😆

Reply:
`snippetTitle`?

Reply:
That could work. Just brainstorming some other ideas

`nodeText`
`nodeLabel`
`textLabel`
`textContentOrAriaLabel` :)

Reply:
I quite like `nodeLabel`, if we're fine with `node.nodeLabel`.

Reply:
I'm down with that. A prefix seems almost necessary to signify that it isn't the literal `label` the node but some derived thing that's label-like, so 👍 

@brendankenny any thoughts on those?

Reply:
> A prefix seems almost necessary to signify that it isn't the literal label the node but some derived thing that's label-like, so 👍

this makes sense to me too, `nodeLabel` sounds good